### PR TITLE
Fixes the Inconsistent placement of frontend action buttons

### PR DESCRIPTION
**Pull request by Pradumn27**
Pull Request for Issue #32250

**Summary of Changes**

This PR changes the placement of buttons from top to bottom in the front-end "module editing" panel so as to make the page alignment consistent with the other front-end editing panels (eg: article editing).

**Testing Instructions**

1, Login to your front-end Joomla website.
2. Click on the edit button next to Main Menu.

**Actual result BEFORE applying this Pull Request**

moduleBefore

**Expected result AFTER applying this Pull Request**

### CODE REVIEW

File: components/com_config/tmpl/modules/default.php [hunk 1]
```diff
@@ -47,23 +47,7 @@
...
-			</button>
-			<button type="button" class="btn btn-danger" data-submit-task="modules.cancel">
-				<span class="icon-times" aria-hidden="true"></span>
-				<?php echo Text::_('JCANCEL'); ?>
-			</button>
-
-			<hr>
-
-			<legend><?php echo Text::_('COM_CONFIG_MODULES_SETTINGS_TITLE'); ?></legend>
+			<legend><b><h2><?php echo Text::_('COM_CONFIG_MODULES_SETTINGS_TITLE'); ?></h2></b></legend>
```
Comment (richard67) [inline components/com_config/tmpl/modules/default.php]: Why `<b><h2>`? This change seems unrelated to the issue, and styling should be done with (s)css.

If it makes sense add a `<h2>` and style this with (s)css, but `<b><h2>` seems weird to me.

But I'd prefer that you revert this change completely.

**Comment by richard67** [comment]
@Pradumn27 In addition to my review comment above I have a question: Have you checked if it needs the same fix at other places, too? The issue #32250 mentions the module edit only as an example, but there might be other places. If possible, please check and report back so we know if this PR fixes the issue completely or if there is something more to be done.

File: components/com_config/tmpl/modules/default.php (hunk 1 above)
Comment (Pradumn27) [reply components/com_config/tmpl/modules/default.php]: Sure, I'll revert that.

**Comment by Pradumn27** [comment]
@richard67 So I had installed the sample data and checked with the different editing options available and the only editing option with a different alignment of buttons was module editing panel.

**Comment by chmst** [comment]
@Pradumn27 Please login on the frontend. Then you get serveral other views with frontend editing for administrators. 
site settings and template settings also have buttons on top. 

Modules have a hr tag which looks not good - I am not sure if this is related to  the changes made here or if it is out of scope and another PR

grafik

**Comment by Pradumn27** [comment]
@chmst I'm a little confused here please do correct me if I'm wrong, so in the issue, it was mentioned that the action buttons(Save, Save and Close, Cancel) of settings accessed on the front-end are inconsistent which actually are. But since you mentioned template and site settings, aren't they the settings in the backend (accessed via administrator).

**Comment by chmst** [comment]
@Pradumn27 yes, this are backend functions, but can be opened also on frontend,: module settings, site settings, template settings .. everything we can open on the site should have the buttons at the same place. 

As it was already said https://github.com/joomla/joomla-cms/issues/32250#issuecomment-800374017 - there could be more edit forms in the frontend.

**Comment by richard67** [comment]
So as this PR doesn't solve all the inconsistencies, I've re-opened issue #32250 since it's not fixed completely.

**Comment by Pradumn27** [comment]
As pointed out by @chmst I have changed the button orientation of site and template settings in the frontend as well. Along with this, I enabled the front-end menu and module editing as well but the menu editing panel is redirected to the administrator panel(aka the backend) and since all the backend functions have the buttons at the top I think changing the menu editing panel's button orientation will make the backend's button orientation inconsistent. Please give your comments.

**Comment by chmst** [comment]
Sorry, no, we will not touch the template atum in backend, only the frontend. Please revert theses changes.

Have a look in the component com_config - there are two forms with buttons in top.

**Comment by Pradumn27** [comment]
@chmst I haven't changed any file in the atum template just the front-end editing forms are changed.

**Comment by ghost** [comment]
@chmst reading this discussion do i understand correct that this PR should not be test now?

**Comment by PhilETaylor** [comment]
I just posted this to the issue: 

Here is the list that needs to be consistent, so choose either Top or bottom and change the others, and fix the icons

| Edit Screen | Location Of Buttons Currently | Buttons | Also |
| ------------- | ------------- | ------------- | ------------- | 
| Article  | Bottom | Save Save&Close Cancel  |
| Module  | Top | Save Cancel Versions |
| Profile  | Bottom | Save Cancel | No tick on save, no cross on cancel
| Template Settings  | Top | Save Cancel |
| Site Settings  | Top | Save Cancel |
| Contact | Bottom | Save Cancel | No cross on cancel  |

**Comment by michwebdev** [comment]
I have tested this item :white_check_mark: successfully on 083e4501313909f9acd98921b8d8f5d4f321c18e<br /><br />Worked as expected.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32703">issues.joomla.org/tracker/joomla-cms/32703</a>.</sub>

**Comment by BertaOctech** [comment]
I have tested this item :white_check_mark: successfully on 083e4501313909f9acd98921b8d8f5d4f321c18e<br /><br />I followed the testing instructions. 

The patch worked as expected<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32703">issues.joomla.org/tracker/joomla-cms/32703</a>.</sub>

**Comment by PhilETaylor** [comment]
>Worked as expected.

>The patch worked as expected

2 tests, but somehow this PR still doesnt address all 6 test cases I have provided....

**Comment by richard67** [comment]
> 2 tests, but somehow this PR still doesnt address all 6 test cases I have provided.... and therefore is not enough to close #32250

@PhilETaylor As far as I can see it addresses the 3 where buttons have to be moved from top to bottom to be consistent with the other 3. But I haven't tested myself.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32703">issues.joomla.org/tracker/joomla-cms/32703</a>.</sub>

**Comment by richard67** [comment]
Will check later on weekend if we need more fixes. I agree to leave the issue open meanwhile.
